src/main/groovy/org/transmartproject/core/dataquery/clinical/ClinicalVariable.groovy

```diff
@@ -84,4 +82,12 @@ public interface ClinicalVariable {
      * or {@link BigDecimal}.
      */
     public final static String NORMALIZED_LEAFS_VARIABLE = 'normalized_leafs_variable'
+
+    /**
+     * The hierarchical path that leads to the concept.
+     *
+     * @return the full path of the concept; path with \ as a separator
+     */
+    public String getConceptPath()
```

Comment:
I just made small improvement: exposed the field that every implementation has to the interface. I don't want to go to deep into rewrite now.